webpack/scenes/ContentViews/Details/Filters/ContentViewFilters.js

```diff
@@ -21,9 +20,10 @@ import { truncate } from '../../../../utils/helpers';
 import ContentType from './ContentType';
 import CVFilterAddModal from './Add/CVFilterAddModal';
 
-const cvFilterUrl = (cvId, filterId) => `/labs/content_views/${cvId}#filters?subContentId=${filterId}`;
+const cvFilterUrl = (cvId, filterId) => `/labs/content_views/${cvId}/filters/${filterId}`;
```

Comment:
This needs updating too:
https://github.com/Katello/katello/blob/6d755b7dff116d611bf6eb5dfc73a3ab0c11cf02/webpack/scenes/ContentViews/Details/Filters/Add/CVFilterAddModal.js#L58